src/index.js

```diff
@@ -419,15 +440,6 @@ function createConfig(options, entry, format, writeMeta) {
 							};
 						},
 					},
-					buble({
-						exclude: 'node_modules/**',
-						jsx: options.jsx || 'h',
-						objectAssign: options.assign || 'Object.assign',
-						transforms: {
-							dangerousForOf: true,
```

Comment:
atm this PR removes `dangerousForOf` behaviour (thus making the output bigger, but more safe), probably as part of this PR we should keep previous semantics, to do the same with babel you have to use this:
```
["@babel/plugin-transform-for-of", { "assumeArray": true }]
```

Reply:
Sure thing, added. No difference in the diffs of the tests, but that is to be expected, as the only file using for-of (`esnext-ts.js`) is currently failing

Reply:
wow, I didn't know Babel had an equivalent for this!